zowex/src/main.rs

```diff
@@ -139,6 +152,75 @@ fn main() -> std::io::Result<()> {
     }
 }
 
+/**
+ * Determine if the command to run is a "zowe daemon" command that we cannot
+ * run from our executable. For such commands, display a zowe NodeJs command
+ * that can be run instead, and exit the current command.
+ *
+ * @param cmd_line_args
+ *      The user-supplied command line arguments to the zowe command.
+ *      Each argument is in its own vector element.
+ */
+fn exit_when_alt_cmd_needed(cmd_line_args: &Vec<String>) {
+    // commands other than daemon commands can be run by our exe
+    if cmd_line_args.len() < 2 {
+        return;
+    }
+    if cmd_line_args[0] != "daemon" {
+        return;
+    }
+    if cmd_line_args[1] != "enable"  &&  cmd_line_args[1] != "disable"  {
+        return;
+    }
+
+    // we can run any of the help requests for the daemon commands
+    if cmd_line_args.len() >= 3 {
+        let third_parm: &str = cmd_line_args[2].as_str();
+        match third_parm {
+            "--help" | "-h" | "--h" | "--help-web" | "--hw" | "--help-examples" => return,
```

Comment:
Having a hardcoded list here of help-related options shared by all commands doesn't seem ideal, but I can't think of a better alternative. As long as we remember to keep this list up-to-date, it should be fine 🙂

Reply:
yeah :-(